pom.xml

```diff
@@ -200,6 +201,18 @@
         <log.specialfilters.acceptonmatch>false</log.specialfilters.acceptonmatch>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>bom</artifactId>
+                <version>${aws.java.sdk.version}</version>
```

Comment:
Note that the SDK is not being used, yet as of now, but the support is now added because the usage is expected based on the S3 PoC.